package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "setup:docs": "./scripts/download-dependencies.sh",
     "setup:native": "./scripts/download-dependencies.sh && ./scripts/setup-react-android.sh",
-    "postinstall": "yarn-deduplicate && expo-yarn-workspaces check-workspace-dependencies && yarn workspace @expo/cli prepare",
+    "postinstall": "yarn-deduplicate && expo-yarn-workspaces check-workspace-dependencies && ([ ! -f node_modules/@react-native/dev-middleware/package.json.orig ] && patch -d node_modules -p2 -f --backup < patches/@react-native+dev-middleware+0.73.4.patch || true) && ([ ! -f node_modules/react-native-gesture-handler/package.json.orig ] && patch -d node_modules -p2 -f --backup < patches/react-native-gesture-handler+2.12.0.patch || true) && ([ ! -f node_modules/react-native-reanimated/package.json.orig ] && patch -d node_modules -p2 -f --backup < patches/react-native-reanimated+3.6.0-nightly-20231031-c56d44fd9.patch || true) && yarn workspace @expo/cli prepare",
```

Comment:
Might be a good idea to move this to a separate script file 🤔

Reply:
maybe we should use patch-package?

Reply:
IIRC, patch-package does not work (maybe due to our monorepo setup). @alanjhughes had some problem with that before.
furthermore, i would prefer not add new dependencies to our repo just for the temporary fixes. we will soon to start updating the vendor modules and should be able to remove these patches then.